client/client.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+VARIANT_ENDPOINT = "http://rest.ensembl.org/vep/human/hgvs/{variant}" 
+
+class VariantAnnotationClient:
+    def __init__(self, variant_endpoint=VARIANT_ENDPOINT):
+        self.variant_endpoint = variant_endpoint
+
+    def parse_variants(self, filename):
```

Comment:
I thought about adding an error handler to the file reading but decided that there's no value to adding custom handling in addition to the default exception throwing.